caffe2/core/nomnigraph/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# ---[ CPU files.
+file(GLOB_RECURSE NOMNI_SRCS *.cc)
+file(GLOB NOMNI_TEST_SRCS test.cc)
+exclude(NOMNI_SRCS "${NOMNI_SRCS}" "${NOMNI_TEST_SRCS}")
+
+add_library(nomnigraph STATIC "${NOMNI_SRCS}")
```

Comment:
`add_library(... OBJECT ...)` may be better if you don't expect any external projects to link against nomnigraph (and thus don't need `libnomnigraph.a`)